gui/src/main/proxy.js

```diff
@@ -0,0 +1,107 @@
+/* eslint standard/no-callback-literal: 0 */
+
+const ElectronProxyAgent = require('electron-proxy-agent')
+const url = require('url')
+const http = require('http')
+const https = require('https')
+
+const log = require('cozy-desktop').default.logger({
+  component: 'GUI:proxy'
```

Comment:
Makes me think that CLI won't have proxy support. Not a priority though. We'll just need to make it clear in case we bring it back.